web/admin/src/Admin.js

```diff
@@ -320,6 +364,26 @@ export default class Admin extends PureComponent {
     return true
   }
 
+  launchAdmin = () => {
+    this.setState({ launchDisabled: true })
+    setTimeout(() => this.setState({ launchDisabled: false }), 2000)
+    openTab(this.adminScreenUrl())
+  }
+
+  adminScreenUrl = () =>
```

Comment:
The name differences between `adminScreenUrl` and `adminScreenUrl` do not tell a reader why their implementations are different.